ts/src/bitget.ts

```diff
@@ -8538,14 +8540,14 @@ export default class bitget extends Exchange {
         //     {"code":"40108","msg":"","requestTime":1595885064600,"data":null}
         //     {"order_id":"513468410013679613","client_oid":null,"symbol":"ethusd","result":false,"err_code":"order_no_exist_error","err_msg":"订单不存在！"}
         //
-        const message = this.safeString (response, 'err_msg');
-        const errorCode = this.safeString2 (response, 'code', 'err_code');
+        const message = this.safeString2 (response, 'err_msg', 'msg');
         const feedback = this.id + ' ' + body;
-        const nonEmptyMessage = ((message !== undefined) && (message !== ''));
+        const nonEmptyMessage = ((message !== undefined) && (message !== '') && (message !== 'success'));
```

Comment:
shouldn't the code have priority over the message? Meaning that we should try to throw first using the code ?

Reply:
but i haven't touched/changed the flow in this PR, can't say why it was done initially this way